users/signals.py

```diff
@@ -0,0 +1,45 @@
+"""Signals for the CusDeb API Users application. """
+
+from urllib.parse import urljoin
+
+from django.dispatch import receiver
+from django.conf import settings
+from django.core.mail import EmailMultiAlternatives
+from django.template.loader import render_to_string
+
+from django_rest_passwordreset.signals import reset_password_token_created
+
+
+@receiver(reset_password_token_created)
+def password_reset_token_created(
+        sender,  # pylint: disable=unused-argument
+        reset_password_token, *_args, **_kwargs
+):
+    """
+      Handles password reset tokens
+      When a token is created, an e-mail needs to be sent to the user
+    """
+    context = {
```

Comment:
Pass `settings.DJANGO_REST_MULTITOKENAUTH_RESET_TOKEN_EXPIRY_TIME` as `token_expiry_time` to the context.